src/steps/devices/index.ts

```diff
@@ -55,6 +55,12 @@ export function createHostAgentProtectsHostRelationship(
         model: device['Hardware Model'],
         serial: device['Serial Number'], // This is needed by Device Schema
         serialNumber: device['Serial Number'],
+
+        ethernetMacAddress: agentEntity.ethernetMacAddress as
```

Comment:
Needed to cast these due to the types in `@jupiterone/integration-sdk-core`.

```ts
declare type EntityPropertyValue = PrimitiveEntityPropertyValue | EntityRawData[];
```

The mapped relationship target entity can't take an `EntityRawData[]` type value...

Reply:
We are able to add arrays with no issues in _mapping on graph-aws 
<img width="624" alt="image" src="https://github.com/JupiterOne/graph-addigy/assets/2650479/7a3f1f60-79c3-4922-b5cb-908e03c1d82d">

Reply:
The `EntityRawData` property is what is causing the issue. Not arrays.